packages/ingest/src/payload/Request.ts

```diff
@@ -126,7 +126,22 @@ export default class Request<T = unknown> {
     if (init.url instanceof URL) {
       this.url = init.url;
     } else if (typeof init.url === 'string') {
-      this.url = new URL(init.url);
+      try {
+        // Try to create URL directly first
+        this.url = new URL(init.url);
+      } catch (e) {
+        // If that fails, try to create with default host
+        try {
+          if (init.url.startsWith('/')) {
+            this.url = new URL(`http://unknownhost${init.url}`);
+          } else {
+            this.url = new URL(`http://unknownhost/${init.url}`);
+          }
+        } catch (e) {
+          // If all else fails, use default URL
+          this.url = new URL('http://unknownhost/');
+        }
+      }
```

Comment:
This conflicts with the latest version. Please pull from head and update that version's logic with your `try catch`. Thanks.